tools/AzureRM.BootStrapper/AzureRM.Bootstrapper.ScenarioTests.ps1

```diff
@@ -0,0 +1,345 @@
+Import-Module -Name AzureRM.Bootstrapper
+$RollupModule = 'AzureRM'
+
+function Remove-InstalledProfile { 
+    $installedProfiles = Get-AzureRmProfile
+    if ($installedProfiles -ne $null)
+    {
+        foreach ($profile in $installedProfiles)
+        {
+            Write-Host "Removing profile $profile"
+            Uninstall-AzureRmProfile -Profile $profile -Force -ErrorAction Stop
+        }
+    }
+}
```

Comment:
Need a verification at the end that profiles were actually uninstalled.  Should throw if not.